web/src/components/MapView.tsx

```diff
@@ -30,11 +30,25 @@ const layerStyle: LayerProps = {
 
 interface MapViewProps {
   featureCollection: FeatureCollection
+  boundingBox?: BBox
+}
+
+const moveViewToBBox = (bBox: BBox, defaultVp: MapViewViewport): MapViewViewport => {
+  const mercatorVp = new WebMercatorViewport(defaultVp)
+  const vp = mercatorVp.fitBounds([
+    [bBox[0], bBox[1]],
+    [bBox[2], bBox[3]]
+  ])
+  return vp
 }
 
 const MapView: React.FunctionComponent<MapViewProps> = (props: MapViewProps): ReactElement => {
-  const [viewport, setViewport] = useState(defaultViewportConfig)
-  const { featureCollection } = props
+  const { featureCollection, boundingBox } = props
+  const [viewport, setViewport] = useState<MapViewViewport>(defaultViewportConfig)
+
+  useEffect(() => {
+    setViewport(boundingBox ? moveViewToBBox(boundingBox, defaultViewportConfig) : defaultViewportConfig)
```

Comment:
@Taggotty 
and yes i love functions :)

Reply:
I think this will clash with #400 right? Not sure but I feel like we need to encapsulate some logic. I.e. have a component calculating the initial viewport and a child rendering the actual MapView and saving the current viewport. That way, no more `useEffects` should be necessary. But maybe I am just too tired and not thinking correctly anymore :D

Reply:
the mistake here is that i have to check if there is a bounding box and then Viewport if not do nothing.
You just have to think about the order doing stuff
1. query param
2. bounding box
3. defaultViewportConfig (which will be deprecated soon when we have all BBoxes)

i don't think it gonna get that complicated but maybe there is a better solution.
lets think about it tommorow